File: Basic_Math/reverse_number/reverseNumber.go
package main

import "fmt"

func reverseNumber(n int) int {
	var lastDigit int = 0
	var reverse int = 0
	for n > 0 {
		lastDigit = n % 10
		n = n / 10
		reverse = (reverse * 10) + lastDigit
	}
	return reverse
}

func main() {
	fmt.Println("reverse number: ", reverseNumber(13456))
}

File: Basic_Math/reverse_number/reverseNumber.js
const reverseNumber = (n) => {
  let lastDigit;
  let reverse = 0;
  while (n > 0) {
    lastDigit = n % 10;
    n = parseInt(n / 10);
    reverse = reverse * 10 + lastDigit;
  }
  return reverse;
};

console.log("reverse number: ", reverseNumber(19999));
